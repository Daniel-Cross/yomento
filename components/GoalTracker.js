import React from "react";
import { Container, StyledText, GoalText } from "../constants/styles";

const GoalTracker = () => {
  return (
    <Container direction="row">
      <Container style={{ paddingRight: 30 }}>
        <GoalText>3</GoalText>
        <StyledText colour="black" style={{ width: 80 }}>
          Weekly Goal
        </StyledText>
      </Container>
      <Container style={{ paddingLeft: 30 }}>
        <GoalText>1</GoalText>
        <StyledText colour="black" style={{ width: 100 }}>
          Completed Activities
        </StyledText>
      </Container>
    </Container>
  );
};

export default GoalTracker;
