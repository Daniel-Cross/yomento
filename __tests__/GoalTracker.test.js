import React from "react";
import renderer from "react-test-renderer";
import GoalTracker from "../components/GoalTracker";

describe("<GoalTracker />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<GoalTracker />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
