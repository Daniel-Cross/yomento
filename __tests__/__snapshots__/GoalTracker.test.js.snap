// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GoalTracker /> renders correctly 1`] = `
<View
  direction="row"
  style={
    Array [
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "space-evenly",
        "paddingBottom": 5,
        "paddingLeft": 5,
        "paddingRight": 5,
        "paddingTop": 5,
        "width": "auto",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "display": "flex",
          "flexDirection": "column",
          "justifyContent": "space-evenly",
          "paddingBottom": 5,
          "paddingLeft": 5,
          "paddingRight": 5,
          "paddingTop": 5,
          "width": "auto",
        },
        Object {
          "paddingRight": 30,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "white",
            "fontSize": 80,
            "fontWeight": "700",
          },
        ]
      }
    >
      3
    </Text>
    <Text
      colour="black"
      style={
        Array [
          Object {
            "color": "white",
            "fontSize": 16,
            "paddingTop": 5,
            "textAlign": "center",
          },
          Object {
            "width": 80,
          },
        ]
      }
    >
      Weekly Goal
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "display": "flex",
          "flexDirection": "column",
          "justifyContent": "space-evenly",
          "paddingBottom": 5,
          "paddingLeft": 5,
          "paddingRight": 5,
          "paddingTop": 5,
          "width": "auto",
        },
        Object {
          "paddingLeft": 30,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "white",
            "fontSize": 80,
            "fontWeight": "700",
          },
        ]
      }
    >
      1
    </Text>
    <Text
      colour="black"
      style={
        Array [
          Object {
            "color": "white",
            "fontSize": 16,
            "paddingTop": 5,
            "textAlign": "center",
          },
          Object {
            "width": 100,
          },
        ]
      }
    >
      Completed Activities
    </Text>
  </View>
</View>
`;
